BootloaderCorePkg/Tools/GenContainer.py

```diff
@@ -220,6 +220,10 @@ def get_pub_key_hash (key, hash_type):
             dh = bytearray (key)[sizeof(PUB_KEY_HDR):]
             dh = dh[:0x100] + dh[0x100:]

```

Comment:
This line is not required

Reply:
Yes, its not required as reverse logic is removed